cmd/minishift/cmd/config/view.go

```diff
@@ -45,7 +47,11 @@ var configViewCmd = &cobra.Command{
 	Short: "Display the properties and values of the Minishift configuration file.",
 	Long:  "Display the properties and values of the Minishift configuration file. You can set the output format from one of the available Go templates.",
 	Run: func(cmd *cobra.Command, args []string) {
-		cfg, err := ReadConfig()
+		confFile := constants.ConfigFile
```

Comment:
WDYT about `minishift config view` showing not only config for the instance, but also information of which global values were used? This would also navigate user about priority of both configs:

```
[agajdosi@dhcp-24-156 ~]$ minishift config view --global
- memory                             : 3gb

[agajdosi@dhcp-24-156 ~]$ minishift config view
- disk-size                             : 30gb

Global values (lower precedence):
- memory                              : 3gb
```

I mean an easy way to see on which values VM is running / will run.

Reply:
@agajdosi I thought it through but since we need to use the global flag to set or unset the config then might be sync with same for the view.

Reply:
> @agajdosi I thought it through but since we need to use the global flag to set or unset the config then might be sync with same for the view.

+1